lib/datasource/remote/api/eos_repository.dart

```diff
@@ -28,12 +28,22 @@ abstract class EosRepository {
   String actionNameAcceptnew = 'acceptnew';
   String actionNameRecover = 'recover';
   String actionNameUnplant = 'unplant';
+  String proposalActionNameDelegate = 'delegate';
+  String proposalActionNameUndelegate = 'undelegate';
 
   // Authorizations
   String permissionActive = 'active';
   String permissionOwner = 'owner';
   String permissionApplication = 'application';
 
+  // Voice scopes
+  static const voiceScopes = [
+    "alliance",
+    "funds.seeds",
+    "milestone",
+    // "referendum", // referendum delegation not working on the contract side at the moment
+  ];
```

Comment:
I'm not fan of static fields Dart allow us to use Top level Objects.
Why not a enum or Just a const List<String> outside the class.

Reply:
Raul is right, this looks like a model, and should be an Enum

Reply:
Not sure why you want to hold this up for this - this is the same as our action names etc... I guess it doesn't need to be static...

Reply:
Ok I changed to List, which makes more sense, same as all the other contract names

It's not an enum.